roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check for usage of deprecated variables
+  debug: msg="{{ __deprecation_header }} {{ item }} is a deprecated variable and will be no longer be used in the next minor release. Please update your inventory accordingly."
+  when:
+  - lookup('env', item) | bool
+  with_items: "{{ __warn_deprecated_vars }}"
+
+# for with_fileglob Ansible resolves the path relative to the roles/<rolename>/files directory
+- name: Assign deprecated variables to correct counterparts
+  include: "{{ item }}"
+  with_fileglob:
+  - "../tasks/__*.yml"
```

Comment:
how about making the association more obvious:

    - "../tasks/deprecations_*.yml"